akka-epl/.gitignore

```diff
@@ -0,0 +1,4 @@
+// you have checked in compiled class files by mistake
+// it's good practice to not do that, easiest is to mark
+// target as ignored
+target/
```

Comment:
I have .gitignore file but on the parent directory. one .gitignore for 3 sub projects.

Reply:
Ah, missed that, and I see that `*/target/` is in there, strange then that the `project/project/target` was checked in, maybe created befor the gitignore-file?